lib/libc/gen/mips/_setjmp.S

```diff
@@ -85,10 +82,7 @@ LEAF(_longjmp)
 	SAVE_GP(CALLFRAME_GP)
 
 	REG_PROLOGUE
-	REG_L		v0, _OFFSETOF_SC_REGS(a0)	# get magic number
 	REG_L		ra, _OFFSETOF_SC_PC(a0)
-	REG_LI		t0, 0xACEDBADE
-	bne		v0, t0, botch		# jump if error
```

Comment:
Might as well remove the code at the `botch` label.

Reply:
done